test/jsonapi/serializer_test.exs

```diff
@@ -124,6 +124,18 @@ defmodule JSONAPI.SerializerTest do
     end
   end
 
+  defmodule CommentView2 do
+    use JSONAPI.View, type: "comment"
+
+    def fields, do: [:text]
+
+    def relationships do
+      # renames "user" data property to "commenter" JSON:API relationship
+      [commenter: {:user1, JSONAPI.SerializerTest.UserView, :include},
+      other: {:user2, JSONAPI.SerializerTest.UserView}]
```

Comment:
Perhaps this test would be clearer if rather than renaming `user2` to `other`, you just passed `user2` through with the default property name `user2`.

Reply:
I'll add a third property that just gets passed through so it is clear just from one test what all the possibilities are -- the reason for the `user2`/`other` case being there is to test that the feature works with a rename but without the `:include` property.